docs_outdoornav_user_manual/features/navigation.mdx

```diff
@@ -0,0 +1,69 @@
+| **Collision Avoidance**     | Collision avoidance is the UGV's ability to detect obstacles and stop/maneuver around the obstacle without any collisions. If `true`, the UGV will detect obstacles and if `false` no obstacles will be detected. Setting to `false` is not recommended and may cause harm to property or to individuals.<br/><br/>Environment Variable: `ENABLE_COLLISION_AVOIDANCE` (Default: `true`). |
+| **Obstacle Avoidance Mode** | When collision avoidance is enabled, the UGV will behave in one of two ways according to the obstacle avoidance mode. If set to `true`, the UGV will perform obstacle avoidance maneuvers, replanning around detected obstacles. If set to `false`, the UGV will slow down to a stop in front of detected obstacles and wait for the obstacle to clear before proceeding.<br/><br/>Environment Variable: `OBSTACLE_AVOIDANCE_MODE` (Default: `true`). |
+| **Continuous Planning**     | The continuous planning feature allows the UGV to continuously monitor whether an obstacle is on the UGV's path and replan around said obstacle smoothly without the need for stopping in front of the obstacle. If disabled, the UGV will come to a full stop in front of an obstacle and then replan around said obstacle.<br/><br/>Environment Variable: `ENABLE_CONTINUOUS_PLANNER` (Default: `true`). |
+| **Path Smoothing**          | The path smoothing feature according to a specified turning radius. The default behaviour will generate point-to-point straight line paths.<br/><br/>Environment Variable: `ENABLE_PATH_SMOOTHER` (Default: `false`). |
+| **Path Shifting**           | The path shifting feature is designed to reduce the oscillation around the initial reference path if the UGV begins to deviate of said reference path. It is particularly useful for the Clearpath Robotics Warthog platform whose tires are incredibly pliable and results in unmodelled effects on the navigation.<br/><br/>Environment Variable: `ENABLE_PATH_SHIFTING` (Default: `false`). |
+| **Constrained Replanning**  | The constrained replanning feature restricts the area in which replanning paths can be generated. For example, if a `REPLANNING_CONSTRAINT` of 3.0 m is used, the UGV will not be allowed to replan a path that drives it more than 3.0 m from the initial path. This replanning area can be shown on the map over the connecting lines between waypoints. See [Constrained Replanning](/docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode#constrained_path) for further details.<br/><br/>Environment Variable: `ENABLE_CONSTRAINED_REPLANNING` (Default: `false`).<br/><br/>Use the `REPLANNING_CONSTRAINT` environment variable to modify the replanning constraint (in meters). |

```

Comment:
Need this to be a relative link

Reply:
Might we want this to be enabled by default to make the system more safe by default?

Reply:
a have a ticket for next sprint to review (ie. remove/update) the navigation features. We hve not updated this page since updating o the behaviour tree and it is out of date.